static/templates/compose_notification.hbs

```diff
@@ -1,5 +1,8 @@
 {{! Content of sent-message notifications }}
-<div class="compose-notifications-content">
-    {{note}} {{#if link_class}}<a href="{{above_composebox_narrow_url}}" class="{{link_class}}" data-message-id="{{link_msg_id}}">{{link_text}}</a>{{/if}}
-    <button type="button" class="out-of-view-notification-close close">&times;</button>
+<div class="above_composebox_banner composebox_banner success {{banner_type}} {{classname}}">
```

Comment:
Maybe `compose_notification` could benefit from a better name. Would `sent_message_notification` be too specific? We could generalize the name if it was used for other things later.